internal/venice-common/src/main/resources/avro/IngestionTaskReport/v1/IngestionTaskReport.avsc

```diff
@@ -16,6 +16,10 @@
       "name": "offset",
       "type": "long"
     },
+    {
```

Comment:
Could you delete `offset`? In-place schema update for IngestionTaskReport is fine since it's used only for IPC.

Reply:
Because we cannot make code and schema changes in one PR because of Enforce Max Lines Added Per File check and also cannot replace offset with position because of compilation failures; we have to add this new field. In next PR we will make code changes, and then in last PR we will remove the offset field from the schema.

PR https://github.com/linkedin/venice/pull/2115 contains all the code+schema changes

Reply:
In this case, it's fine to make both changes in single PR and use VALIDATION_OVERRIDE to bypass as this schema is used only for IPC